akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -384,9 +386,12 @@ final class ActorMaterializerSettings private (
   /**
    * Each asynchronous piece of a materialized stream topology is executed by one Actor
    * that manages an input buffer for all inlets of its shape. This setting configures
-   * the initial and maximal input buffer in number of elements for each inlet.
+   * the default for initial and maximal input buffer in number of elements for each inlet.
+   * This can be overridden for individual parts of the
+   * stream topology by using [[akka.stream.Attributes#inputBuffer]].
```

Comment:
That it is set together with the boundaries rather than separately for a larger part of the graph? I'm not sure. But we could definitely mention the convenience `async()` method.

Reply:
Note though that it is used by a slew of stages that is not necessarily an async boundary (see FIXME below)